src/utils.ts

```diff
@@ -191,6 +192,29 @@ export function logLabeledWarning(
   logger[type](clc.yellow.bold(`${WARNING_CHAR}  ${label}:`), message, data);
 }
 
+/**
+ * Log an info statement with a gray bullet at the start of the line.
+ */
+export function logDebug(
+  message: string,
+  type: LogLevel = "debug",
+  data: LogDataOrUndefined = undefined
+): void {
+  logger[type](clc.blue.bold(`${DEBUG_CHAR} `), message, data);
+}
+
+/**
+ * Log an info statement with a gray bullet at the start of the line.
+ */
+export function logLabeledDebug(
+  label: string,
+  message: string,
+  type: LogLevel = "debug",
+  data: LogDataOrUndefined = undefined
+): void {
+  logger[type](clc.blue.bold(`${DEBUG_CHAR}  ${label}:`), message, data);
+}
```

Comment:
What's the utility of these two functions? AFAIK, what will happen here is that these will make it into the debug log (output only if `--debug` is provided or only into `firebase-debug.log`). That's the same as just using `logger.debug` in the end. I don't see the value add here...

Reply:
Makes sense - I didn't see the `logger.debug` 👍